pkg/pureutils/purefa_rest2x.go

```diff
@@ -64,6 +64,24 @@ func ListAllPodsFromFA(faClient *flasharray.Client) ([]flasharray.PodResponse, e
 	return pods, nil
 }
 
+// GetCompleteVolumeNameFromFA returns complete volume name from FA
+func GetCompleteVolumeNameFromFA(faClient *flasharray.Client, volumeName string) (string, error) {
+	vols, err := ListAllVolumesFromFA(faClient)
```

Comment:
it could be unneccessary to list all volumes, add a new method to fetch only one volume

Reply:
i have written this because as volume name actually defers in FA, it will be different when we have pod-realm config, pod config, and no realm-pod config so we need to list volumes and from that filter the volume we require

Reply:
added a new function which takes volume name and call api call volumes?names=<volume name>